cirq-aqt/cirq_aqt/aqt_sampler.py

```diff
@@ -120,7 +176,7 @@ def _send_json(
 
```

Comment:
The docstring needs an update (references PUT requests).

Reply:
Just caught this too, thanks!